### ISSUE TITLE
Move shutter further when it's not fully closed

### ISSUE THREAD

**Issue by ghilios** [nexdome/ASCOM#41]
Sometimes the shutter motor jumps a few rail teeth and falls short of the battery during parking. When I'm physically present I engage the motor control switch manually, but how would I do this if I'm remote? There doesn't seem to be a "move" command for the shutter, and close (CLS) won't go any further than where it thinks is currently the zero position which isn't quite fully closed.

**Comment by Josh-Jones-76** [comment]
Yes, I agree, this is someting I also noticed if remoting from a video camera and seeing it not reach after it skipped a few.

I think another option here is when it powercycles, the shutter being partiall open perhaps, it doesn't move since it's set to be closed on boot loading.  Manually moving would be great to get it to the closed or open position and then start from there.  

Calibrating on the magnets would be nice after powercycles as well, so it knows it's at 0 or 46000 on the @PWS.

**Comment by Josh-Jones-76** [comment]
Here is a good example, wake up and it's stuck here.  This may have skipped a tooth, and I can see hardware issue not being a concern, but the other scenario is power cycle or loss and then it won't ever shut or open.  

Thought maybe a find home/closed button might be nice just like rotator has, but might not work if you have a situation where it doesn't see magnet?  Maybe it would go to far?  I feel like a 'find closed' or 'find open' to reset the position would be nice feature to force it past steps, but could run risk of hardware running to far and damaging things perhaps?

Only other thing I could think of here is a manual shutter open/close feature like rotator, where we could close/open shutter in steps or percentages.  be more handy than opening and consoling to it, rewriting position with @PWS,xxxx  and then @CLS until you get it.

![nexdome](https://user-images.githubusercontent.com/89028102/130642258-02c7f406-1c37-4ba1-973f-342dbb4277f7.jpg)

**Comment by ghilios** [comment]
I agree. I don't think we need anything particularly involved, but the ability to manually move the shutter up and down even beyond the limits, as well as setting a "home"/"closed" position would enable better remote support. Otherwise I have to show up in person to fix it.

@NameOfTheDragon - do you think there's enough space on the image to support an additional command or 2 for moving the shutter manually? I'd be up for contributing some code if possible, but I couldn't get the firmware to compile the last time I tried.
